lib/kubernetes-deploy/runner.rb

```diff
@@ -69,39 +69,24 @@ class Runner
 
     def self.with_friendly_errors
       yield
-    rescue FatalDeploymentError => error
-      KubernetesDeploy.logger.fatal <<-MSG
-#{error.class}: #{error.message}
-  #{error.backtrace && error.backtrace.join("\n  ")}
```

Comment:
I removed the backtrace because in our experience seeing it leads people to assume the deploy itself blew up and the failure is not their own fault.